src/index.ts

```diff
@@ -3,38 +3,59 @@
 
 // Blind RSA draft 14
 // https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-rsa-blind-signatures-14
-import { BlindRSA, PrepareType } from './blindrsa.js';
+import { BlindRSA, PrepareType, type BlindRSAParams } from './blindrsa.js';
 
-export type { BlindRSA };
+export { BlindRSA, type BlindRSAParams };
 
-export const SUITES = {
+export const Params: Record<string, BlindRSAParams> = {
+    RSABSSA_SHA384_PSS_Randomized: {
```

Comment:
you may want to document the use of `RSABSSA`. you mention `RSABSSA, which is the actual name of the scheme`, which might point to a reference

Reply:
fun fact: rfc9474 was just published today.
so all documentation links need to be updated.

Reply:
good news and more work

Reply:
[my understanding of the publication state is it got assigned a number but needs approvals for all the authors]